experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { TypedEventEmitter } from "@fluidframework/common-utils";
+import { Container } from "@fluidframework/container-loader";
+import { ICriticalContainerError } from "@fluidframework/container-definitions";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import { IEvent } from "@fluidframework/common-definitions";
+import { LoadableObjectClass, LoadableObjectRecord } from "./types";
+import { RootDataObject } from "./rootDataObject";
+interface IFluidContainerEvents extends IEvent {
+    (event: "connected", listener: (clientId: string) => void): void;
+    (event: "dispose" | "disconnected", listener: () => void): void;
+}
+
+interface IFluidContainer {
+    close(error?: ICriticalContainerError): void;
+    readonly closed: boolean;
```

Comment:
Few thoughts:
1. We shouldn't take an error arg for sure
2. We should pick one of "dispose" vs. "close" and use it consistently (currently this has `close()`, `closed`, and `"dispose"` event)
3. My preference is for "dispose" since its meaning is more obvious without reading documentation.  "Closing a container" sounds like it could mean something special rather than just destruction, and also sounds like it might be paired with an "open"

```suggestion
    dispose(): void;
    readonly disposed: boolean;
```